src/executables/util_admin.c

```diff
@@ -895,7 +895,7 @@ static GETOPT_LONG ua_Flashback_Option[] = {
 
 static UTIL_ARG_MAP ua_Memmon_Option_Map[] = {
   {OPTION_STRING_TABLE, {0}, {0}},
-  {MEMMON_MODULE_S, {ARG_INTEGER}, {(void *) -2}},
+  {MEMMON_MODULE_S, {ARG_INTEGER}, {(void *) -2097573308}},
```

Comment:
We need a comment if something is not self-explanatory. What's it for?

Reply:
Getting integer option value can't differentiate about user input value or default value. It can make some error case. So I set default value with magic number to pass this case.

I add it to comment.